add benchmark for stumpless_get_prival_string

amitansu1024 opened the pull request:
Adds benchmarking for the function stumpless_get_prival_string. fixes #431

amitansu1024 committed:
add benchmark stumpless_get_prival_string (#431)

Adds benchmarking for the function stumpless_get_prival_string.

goatshriek commented on test/performance/prival.cpp:
```diff
@@ -44,4 +45,25 @@ static void PrivalFromString(benchmark::State& state) {
+      result = stumpless_get_prival_string( prival );
+    }
+  }
+  free( ( void * ) result );
```
This call to `free` will need to move to inside both loops to avoid a memory leak in this test.

goatshriek reviewed (changes requested):
Thanks for putting this together! I believe two changes will be enough for this to be accepted: moving `free` as noted below, and adding the header for `free` as well.

I'm unsure why the locale CI builds are failing but not others. Let's see if the other changes resolve this failure as well.

amitansu1024 committed:
Add required headers and move free inside loops

Adds required headers for free function. Move free inside loops to
avoid memory leaks in test.

goatshriek commented:
Thanks again! I've identified the issue affecting some of the builds - this will get merged once that makes it in so that this change doesn't falsely appear to cause failures.
